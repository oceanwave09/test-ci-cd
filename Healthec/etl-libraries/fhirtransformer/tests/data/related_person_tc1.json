{
    "firstname": "Michael",
    "lastname": "Jackson",
    "gender": "male",
    "dob": "1967-07-01",
    "active": "true",
    "street_address_1": "132, Erewhon St",
    "city": "Kingston",
    "state": "New York",
    "zip": "12401",
    "phone_mobile": "(03) 3410 5613",
    "email": "michael.jackson@gmail.com",
    "patient_id":"f2b8db7e-c0a4-40d2-a245-93ea68699394",
    "subscriber_id":"48232",
    "relationship_code": "other",
    "relationship_display": "Other",
    "period_start_date": "2023-01-01",
    "period_end_date": "2023-12-31"
  }