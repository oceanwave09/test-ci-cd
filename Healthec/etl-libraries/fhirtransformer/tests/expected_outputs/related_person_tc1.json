{
    "resourceType": "RelatedPerson",
    "identifier": [
      {
        "type": {
          "coding": [
            {
              "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
              "code": "SN",
              "display": "Subscriber Number"
            }
          ]
        },
        "value": "48232"
      }
    ],
    "name": [
      {
        "use": "official",
        "family": "Jackson",
        "given": [
          "Michael"
        ]
      }
    ],
    "active": true,
    "patient": {
      "reference": "Patient/f2b8db7e-c0a4-40d2-a245-93ea68699394"
    },
    "relationship": {
      "coding": [
        {
          "code": "other",
          "display": "Other"
        }
      ]
    },
    "telecom": [
      {
        "system": "phone",
        "use": "mobile",
        "value": "(03) 3410 5613"
      },
      {
        "system": "email",
        "value": "michael.jackson@gmail.com"
      }
    ],
    "gender": "male",
    "birthDate": "1967-07-01",
    "address": [
      {
        "line": [
          "132, Erewhon St"
        ],
        "city": "Kingston",
        "state": "New York",
        "postalCode": "12401"
      }
    ],
    "period": {
      "start": "2023-01-01",
      "end": "2023-12-31"
    }
  }